docs/getting-started.md

```diff
@@ -26,6 +26,18 @@ A quick start guide to get KubeVirt up and running inside Vagrant.
     sudo dnf install golang
 ```
 
+**Note:** Some code within k8s.io/client-go and k8s.io/apimachinery uses
+features from the Go standard libaries introduced in version 1.8. This affects
+Fedora 25 as it ships with Go 1.7.5.
```

Comment:
I don't think we need to call out Fedora 25 particularly here - it affects basically every distro that isn't a very recent release, or able to pull in GO libraries via some add on repository. It is sufficient to document that you need to be running a distro with Go 1.8.  I'd suggest we just have a section right at the start of the file where we list any pre-requisites we have for development